middleware/packet-forward-middleware/packetforward/migrations/v3/migrate.go

```diff
@@ -0,0 +1,55 @@
+package v3
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v8/packetforward/types"
+
+	transfertypes "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+)
+
+// Migrate migrates the x/packetforward module state from the consensus version
+// 2 to version 3
+func Migrate(
```

Comment:
Not a very standard migration here since it affects other modules instead of the current module state, open to feedback! Alternatively, resolving state in it in an upgrade handler without bumping consensus versions could be an option